examples/shardingsphere-sample/shardingsphere-example-generator/src/main/java/org/apache/shardingsphere/example/generator/scenario/ExampleScenario.java

```diff
@@ -37,6 +38,20 @@
      * @return resource template map
      */
     Map<String, String> getResourceTemplateMap();
+
+    /**
+     * Get java class path set.
```

Comment:
Please change java from `path set` to `paths`

Reply:
done.